install/CMakeLists.txt

```diff
@@ -79,11 +80,41 @@ endfunction()
 #
 get_target_property(include_dirs zenohc::lib INTERFACE_INCLUDE_DIRECTORIES)
 foreach(dir ${include_dirs})
-    install(DIRECTORY "${dir}/" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+    install(DIRECTORY "${dir}/" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR} COMPONENT headers)
 endforeach()
 
 #
 # Install lib files 
 #
 install_zenohc_lib("Release;RelWithDebInfo;MinSizeRel;None" "RELEASE" zenohc)
 install_zenohc_lib("Debug" "DEBUG" zenohc)
+
+#
+# Configure CPack
+#
+set(CPACK_PACKAGE_DIRECTORY "${CMAKE_BINARY_DIR}/packages")
+set(CPACK_PACKAGE_CHECKSUM MD5)
+set(CPACK_PACKAGE_VENDOR "The Eclipse Foundation")
+set(CPACK_PACKAGE_VERSION_MAJOR ${PROJECT_VERSION_MAJOR})
+set(CPACK_PACKAGE_VERSION_MINOR ${PROJECT_VERSION_MINOR})
+set(CPACK_PACKAGE_VERSION_PATCH ${PROJECT_VERSION_PATCH})
+set(CPACK_COMPONENT_LIB_DESCRIPTION "The C client library for Eclipse zenoh")
+set(CPACK_COMPONENT_DEV_DESCRIPTION "${CPACK_COMPONENT_LIB_DESCRIPTION} - cmake files")
+set(CPACK_COMPONENT_HEADERS_DESCRIPTION "${CPACK_COMPONENT_LIB_DESCRIPTION} - header files")
+set(CPACK_COMPONENT_LIB_GROUP "lib")
+set(CPACK_COMPONENT_HEADERS_GROUP "dev")
+set(CPACK_COMPONENT_DEV_GROUP "dev")
+set(CPACK_COMPONENT_HEADERS_DEPENDS lib)
+set(CPACK_COMPONENT_DEV_DEPENDS lib)
+
+set(CPACK_GENERATOR ZIP)
+# set package name to format <name>-<version>.<extension>
+# platform name doesn't matter for header-only library
+set(CPACK_PACKAGE_NAME zenoh-c)
+if(NOT CPACK_PACKAGE_VERSION)
+  set(CPACK_PACKAGE_VERSION ${PROJECT_VERSION}) 
```

Comment:
The package version is also wrong, it needs to use the debian version instead. Like in here: https://github.com/eclipse-zenoh/zenoh-pico/blob/main/CMakeLists.txt#L554